sdlf-cicd/template-cicd-team-repos.yaml

```diff
@@ -34,6 +34,24 @@ Resources:
                   - codecommit:PutRepositoryTriggers
                   - codecommit:UpdateRepositoryDescription
                   - codecommit:UploadArchive
+                  - codecommit:AssociateApprovalRuleTemplateWithRepository
+                  - codecommit:CreateApprovalRuleTemplate
+              - Resource: !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:sdlf-repo-notifications-*
+                Effect: Allow
+                Action:
+                  - SNS:GetTopicAttributes
+                  - SNS:CreateTopic
+                  - SNS:Subscribe
+                  - SNS:SetTopicAttributes
+                  - SNS:ListSubscriptionsByTopic
+              - Resource: !Sub arn:aws:events:${AWS::Region}:${AWS::AccountId}:rule/sdlf-pr-created-*
+                Effect: Allow
+                Action:
+                  - events:DescribeRule
+                  - events:DeleteRule
+                  - events:PutRule
+                  - events:PutTargets
+                  - events:RemoveTargets
               - Resource: "*"
```

Comment:
Also related to the Multiple SNS feature by Ricardo, we need to add an `InvokeFunction` permission here:
```
              - Resource: !Sub arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:sdlf-cr-sns-topic-endpoints-subscription
                Effect: Allow
                Action:
                  - lambda:InvokeFunction
```

Reply:
I'll look into it